dist/angular-patternfly.js

```diff
@@ -1670,6 +1683,7 @@ angular.module('patternfly.charts').directive('pfHeatmap', ["$compile", "$window
        }
 
        $scope.data = {
+         dataAvailable: true,
```

Comment:
> We added the flag in the angular patternfly directive so the application could differentiate between an empty dataset vs data unavailable.

@simon3z if you look into the [containers dashboard](https://github.com/zeari/manageiq/blob/396befd12a5ec4b71969cfb32c93a9235c31dcfa/app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js#L129-129) youll see that a `dataAvailable` flag is the current standard with these patternfly directives.